app/controllers/flatirons/saas/subscriptions_controller.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module Flatirons::Saas
+  class SubscriptionsController < ApplicationController
```

Comment:
this will be moved to a generator i guess

Reply:
we will use it internally to generate the routes to users get subscriptions.
GET /flatrions-saas/subscriptions